templates/zerver/help/join-a-zulip-organization.md

```diff
@@ -0,0 +1,40 @@
+# Join a Zulip Organization
+
+An **organization**  is a private chamber hosted on Zulip with its own users,
+ streams, customizations, etc. Administrators manage the organization,
+ controlling the specific settings of the organizations
+ that they own.
+
+1. To join a Zulip organization, first navigate to the URL of the Zulip server.
+
+2. Click the register button in the top right hand corner of the screen.
+
+    ![Register button](/static/images/help/register-button.png)
+
+3. You will be taken to the register page and prompted to **Enter your
+ work email address**. Enter your work email address and click **Sign Up**.
```

Comment:
Make sure `**Enter your work email address**` is on the same line without line breaks to prevent rendering errors.